setup.cfg

```diff
@@ -27,7 +27,8 @@ install_requires =
 [options.extras_require]
 fastapi = 
     fastapi
-	uvicorn
+	pydantic-settings
```

Comment:
So this forces to use `pydantic` v2:

https://github.com/pydantic/pydantic-settings/blob/4f794c7442a3c71d6d3280c888cdd8c288350543/pyproject.toml#L42-L45

Reply:
Do you think we should state the dependency explicitly here ?

Reply:
Yes maybe, solving dependency issues quickly becomes a nightmare.